### PR TITLE
Update sunpy test for 2.1

### PR DESCRIPTION
**Pull request by Cadair**

### CODE REVIEW

**Comment by astrofrog** [comment]
Thanks! There is actually another test or two with the same code-ish elsewhere, search for ``heliographic_observer`` - could you fix these too?

File: setup.cfg [hunk 1]
```diff
@@ -30,7 +30,7 @@ test =
     pytest-doctestplus
 testall =
     shapely;sys_platform!='win32' and python_version!='3.8'
-    sunpy;platform_machine!='i686'
+    sunpy[map];platform_machine!='i686'
```
Comment (astrofrog) [inline setup.cfg:33]: Can you add ``>=2.1`` here?

Comment (Cadair) [reply setup.cfg:33]:
```suggestion
    sunpy[map]>=2.1;platform_machine!='i686'
```

**Comment by astrofrog** [comment]
This looks good and the CI failures are unrelated I think

**Pull request merged**